### ISSUE TITLE
GET /callsets/{id} fails when passed ID returned from POST /callsets/search

### ISSUE THREAD

**Issue by hjellinek** [ga4gh/server#598]
This is from `CallsetsSearchResponseCheckIT.getCallSetWithValidIDShouldSucceed'.  The rough outline is this:

1. Search `VariantSets` (`POST /variantsets/search`)
1. Store the first `VariantSet` returned in a local variable
1. Search for all `CallSets` (`POST /callsets/search`) with that variant set ID
1. Store the first `CallSet` returned in a local variable - `theCallSet` and ID in `theId`
1. Call `GET /callsets/{theId}`

Expected result: a `CallSet` object.

Actual result:  HTTP status `500`: `Internal Server Error`.
The ref server reports the cause is `KeyError`.

--------
### Stack trace:

```
ERROR in app [/Users/hjellinek/Library/Python/2.7/lib/python/site-packages/flask/app.py:1423]:
Exception on /callsets/YnJjYTE6MWtnUGhhc2UzOkhHMDAwOTY= [GET]

 Traceback (most recent call last):
  File "/Users/hjellinek/Library/Python/2.7/lib/python/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/hjellinek/Library/Python/2.7/lib/python/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/hjellinek/Work/server/ga4gh/frontend.py", line 535, in getCallset
    id, flask.request, app.backend.runGetCallset)
  File "/Users/hjellinek/Work/server/ga4gh/frontend.py", line 369, in handleFlaskGetRequest
    return handleHttpGet(id_, endpoint)
  File "/Users/hjellinek/Work/server/ga4gh/frontend.py", line 285, in handleHttpGet
    responseStr = endpoint(id_)
  File "/Users/hjellinek/Work/server/ga4gh/backend.py", line 558, in runGetCallset
    callSet = variantSet.getCallSet(id_)
  File "/Users/hjellinek/Work/server/ga4gh/datamodel/variants.py", line 346, in getCallSet
    return self._callSetIdMap[callSetId]
 KeyError: 'YnJjYTE6MWtnUGhhc2UzOlluSmpZVEU2TVd0blVHaGhjMlV6T2toSE1EQXdPVFk9'
 127.0.0.1 - - [01/Sep/2015 10:04:32] "GET /callsets/YnJjYTE6MWtnUGhhc2UzOkhHMDAwOTY= HTTP/1.1" 500 -
```

**Comment by jeromekelleher** [comment]
Nice bug report @hjellinek --- these compliance tests are really giving me a warm fuzzy feeling!

**Comment by dcolligan** [comment]
Yes, I agree.  These bug reports are really easy to diagnose with all of this data, @hjellinek ... fix coming shortly...

**Comment by dcolligan** [comment]
Fixed in #602
